README.md

```diff
@@ -250,6 +250,22 @@ let noWarning :String = "" // No warning about colons immediately after variable
 let hasWarning :String = "" // Warning generated about colons immediately after variable names
 ```
 
+Including the `all` keyword will disable all rules until the linter sees a matching enable comment:
+
+`// swiftlint:disable all [<rule1> <rule2>...]`
```

Comment:
should we remove the `[<rule1> <rule2>...]`?

Reply:
I would leave it since it's still possible to exclude single other rules additionally, then disable the all rule and have the single other rules still be in effect.

Reply:
Since we already cover disabling individual rules above, I'd just remove them here to make it clearer to understand. I don't expect people to use `all` with other rule identifiers commonly.

Reply:
Makes sense